src/zkevm_specs/state_circuit.py

```diff
@@ -80,6 +79,7 @@ class Row(NamedTuple):
                       FQ,FQ,FQ,FQ,FQ,FQ,FQ,FQ]
     value: FQ
     committed_value: FQ
+    initial_value: FQ
```

Comment:
I was under the impression that `committed_value` and `initial_value` are the same concept.

There's an inconsistency in our code/specs.  I think CommittedValue and InitValue are the same.  In the circuit it's "initial_value", in the spec tables it's "InitialValue" and in the python specs it's "committed_value".

I think it makes sense to call the column "initial_value", and the meaning of initial_value for state_trie values is committedValue (because it was the value committed in the previous state trie).

So I think instead of adding a column named `initial_value` we should rename `committed_value` to `initial_value`.

---

The column that may be missing that is discussed in https://github.com/privacy-scaling-explorations/zkevm-specs/issues/202 would be `previous_vale`, but we can figure that out in another PR.